Add clicker training plugin

pyonium opened the pull request:
Adds a clicker training plugin that plays a dog-clicker sound every time you level up a skill. Classical conditioning!

Nightfirecat commented:
[I'm fairly sure Adam doesn't have sole copyright of the code you wrote.](https://github.com/pyonium/clicker-training-osrs/blob/a922393991e7cf91da5e9afeed9c507c561dde9e/LICENSE#L3) You should update this file accordingly. (particularly including the plugins you borrowed code from)
Also I don't know the source of this audio file (given it is so generic), I presume it is licensed in a way which we can distribute it?

pyonium commented:
Should be fixed now! Updated the license to add the original authors of the borrowed code.

Sound is from https://freesound.org/people/dnewtonjr/sounds/231559/ under Creative Commons 0, modified. Should be good.
